### PR TITLE
Closing multiprocessing pool to avoid too many open files

### PR DESCRIPTION
**Pull request by heliy**
Hello. I meet the "too many open files" expection when I using 100 threads in a Linux machine. I fix it and eager to pull into main repo. :)

### CODE REVIEW

**Comment by j6k4m8** [comment]
This is excellent, thank you for the fix, @heliy!

File: intern/service/boss/v1/volume.py [hunk 1]
```diff
@@ -277,6 +277,7 @@ def get_cutout(
                         # TODO: kwargs
                     )
                  for b in blocks])
+                pool.close()
```
Comment (movestill) [inline intern/service/boss/v1/volume.py:280]: Do you mind making a minor change so we use a context for `pool` on line 271, instead?
```python
with multiprocessing.Pool(processes=parallel) as pool:
```

**Review by movestill — CHANGES_REQUESTED** [review]
Nice catch! I think it'll be a little more robust if you make the requested change. Thanks!

File: intern/service/boss/v1/volume.py (hunk 1 above)
Comment (heliy) [reply intern/service/boss/v1/volume.py:280]: That is good！I will do that.

**Review by movestill — APPROVED** [review]

**Pull request merged**